includes/class-coblocks-block-assets.php

```diff
@@ -160,14 +160,6 @@ public function editor_assets() {
 		$name       = 'coblocks-extensions';
 		$filepath   = 'dist/' . $name;
 		$asset_file = $this->get_asset_file( $filepath );
-		$rtl        = ! is_rtl() ? '' : '-rtl';
-
-		wp_enqueue_style(
-			$name,
-			COBLOCKS_PLUGIN_URL . 'dist/style-' . $name . $rtl . '.css',
-			array(),
-			$asset_file['version']
-		);
```

Comment:
We are removing the RTL logic here? Why? We have RTL files generated and available to enqueue.

Reply:
No, it's because the $rtl variable is calculated earlier already

Reply:
Oh you know what. I think I see why you did that now. We import that block_assets function and this editor_assets function.

Reply:
As a side note - We really need to refactor this file. We have this redundant import going on and it is very confusing to see what exactly is happening.

https://github.com/godaddy-wordpress/coblocks/blob/dcfae001f4b97cc050cda6d3e438b8e9e945631b/includes/class-coblocks-block-assets.php#L47-L49

Reply:
@AnthonyLedesma totally agree!